packages/api/test/test-suites/filters.ts

```diff
@@ -124,12 +124,6 @@ export function runFilterTests(
       });
     });
 
-    it("should return the correct results when filtering by a 'null' rollup", async () => {
```

Comment:
why did you remove this test case?

Reply:
I removed it while testing and forgot to re-add it. I’ve just pushed a commit to add it back.